src/components/Navigator/NavigatorCard.vue

```diff
@@ -28,6 +32,18 @@
         @keydown.up.exact.capture.prevent="focusPrev"
         @keydown.down.exact.capture.prevent="focusNext"
       >
+        <Reference
+          v-if="technology"
+          :id="INDEX_ROOT_KEY"
+          :url="technologyPath"
+          class="technology-title"
+          @click.alt.native.prevent="toggleAllNodes"
+        >
+          <h2 class="card-link">
+            {{ technology }}
+          </h2>
+          <Badge v-if="isTechnologyBeta" variant="beta" />
```

Comment:
Do we also care about deprecation here?

Reply:
It seems that in `main` we are only applying the beta badge 🤔  but we can look into adding the technology badge in another PR.